libcloud/__init__.py

```diff
@@ -75,3 +94,40 @@ def _init_once():
             paramiko.common.logging.basicConfig(level=paramiko.common.DEBUG)
 
 _init_once()
+
+
+class DriverType:
```

Comment:
Good idea to be consistent and inherit from object even if it's just used as an ENUM class.